specs/host_list_view.adoc

```diff
@@ -0,0 +1,148 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of hosts discovered by Tendrl
+
+Display list of hosts discovered by Tendrl, whether they currently associated with a cluster or not. The list should have functionality for 
+
+* filtering by name, cluster, role and status.
+* sorting by name.
+
+Host list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of hosts discovered by Tendrl along with their details.
+
+The details shown for each host in UI is -
+
+* Host Name
+* Status
+* IP Address
+* Storage Usage
+* Utilisation
+* Memory Usage
+* Cluster Name
+* Role
+* Alert Count
+
+== Use Cases
+
+Users can see the list of host present in Tendrl. They can also 
+
+* filter by name, cluster, role and status.
+* sort by name.
+
+As per the API support, users can 
+
+* filter by name and cluster
+* sort by name
+
+== Proposed change
+
+* Create a view for listing host details.
+* For showing storage usage, utilisation and memory usage, use donut chart component.
+* Implement client side filer and sorting functionality using angular's filter and orderby components.
+* Send polling call after 2 minutes to get the updated data.
+* Show total count of hosts present discoverable by Tendrl.
+
+=== Alternatives
+
+Generate list component can be created and used for listing the host. But because of its complexity and time constraint, we will pick this up later.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
+
+/api/1.0/GetNodeList - fetch the list of host
+
+API support present for -
+
+* Host Name
+* IP Address
+* Storage Usage
+* Utilisation
```

Comment:
The same as in previous comment, what utilization is meant here. Probably CPU but it's not clear.